query/src/main/java/org/infinispan/search/mapper/work/impl/SearchIndexerImpl.java

```diff
@@ -47,9 +67,9 @@ public CompletableFuture<?> addOrUpdate(Object providedId, String routingKey, Ob
          return CompletableFuture.completedFuture(null);
```

Comment:
I missed this before, all of these references should be replaced with `CompletableFutures.completedNull()`